src/iexamine_actors.h

```diff
@@ -0,0 +1,41 @@
+#include "iexamine.h"
+
```

Comment:
Probably can resolve GCC 7 build error:
```suggestion
#include <memory>
```